src/css/bootstrap-graphdb-theme.css

```diff
@@ -1412,6 +1412,7 @@ footer.footer .container-fluid.main-container p {
 -------------------------------------------------------------*/
 .nav-tabs {
     font-weight: 400;
+    display: inline-block;
```

Comment:
I don't quite like setting this here in the theme because it would affect all nav-tabs, but I think it's OK for now